extensions/amp-kaltura-player/0.1/amp-kaltura-player.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isLayoutSizeDefined} from '../../../src/layout';
+import {loadPromise} from '../../../src/event-helper';
+import {addParamsToUrl} from '../../../src/url';
+import {dashToCamelCase} from '../../../src/string';
+import {setStyles} from '../../../src/style';
+
+class AmpKaltura extends AMP.BaseElement {
+
+    /** @override */
+    createdCallback() {
+      this.preconnect.url('https://cdnapisec.kaltura.com');
+    }
+
+    /** @override */
+    isLayoutSupported(layout) {
+      return isLayoutSizeDefined(layout);
+    }
+
+    /** @override */
+    buildCallback() {
+      if (!this.getPlaceholder()) {
+        this.buildImagePlaceholder_();
+      }
+    }
+
+    /** @override */
+    layoutCallback() {
+      const width = this.element.getAttribute('width');
+      const height = this.element.getAttribute('height');
+      const partnerid = AMP.assert(
+        this.element.getAttribute('data-partner'),
```

Comment:
Line continuations like this should be indented 4 spaces. Sorry for not catching this in the linter.